core/src/main/java/org/polypheny/db/catalog/impl/allocation/PolyAllocRelCatalog.java

```diff
@@ -0,0 +1,295 @@
+        AllocationColumn column = columns.get( Pair.of( placementId, columnId ) ).toBuilder().placementType( placementType ).build();
+        columns.put( Pair.of( placementId, columnId ), column );
+        change();
+    }
+
+
+    @Override
+    public AllocationPartitionGroup addPartitionGroup( long tableId, String partitionGroupName, long namespaceId, PartitionType partitionType, long numberOfInternalPartitions, boolean isUnbound ) {
+        long id = idBuilder.getNewGroupId();
+        if ( log.isDebugEnabled() ) {
+            log.debug( "Creating partitionGroup of type '{}' with id '{}'", partitionType, id );
+        }
+
+        AllocationPartitionGroup partitionGroup = new AllocationPartitionGroup(
+                id,
+                partitionGroupName,
+                tableId,
+                namespaceId,
+                0,
+                isUnbound );
+
+        partitionGroups.put( id, partitionGroup );
+        return partitionGroup;
+    }
+
+
+    @Override
+    public void deletePartitionGroup( long groupId ) {
+        partitionGroups.remove( groupId );
+    }
+
+
+    @Override
+    public AllocationPartition addPartition( long tableId, long namespaceId, long groupId, @Nullable String name, boolean isUnbound, PlacementType placementType, DataPlacementRole role, List<String> qualifiers, PartitionType partitionType ) {
+        long id = idBuilder.getNewPartitionId();
+        if ( log.isDebugEnabled() ) {
+            log.debug( "Creating partition with id '{}'", id );
+        }
+
+        AllocationPartition partition = new AllocationPartition(
+                id,
+                namespaceId,
+                tableId,
+                groupId,
+                placementType,
+                name,
+                role,
+                isUnbound,
+                qualifiers,
+                partitionType );
+
+        partitions.put( id, partition );
+        change();
+        return partition;
+    }
+
+
+    @Override
+    public void deletePartition( long partitionId ) {
+        partitions.remove( partitionId );
+    }
+
+
+    @Override
+    public void addPartitionProperty( long tableId, PartitionProperty partitionProperty ) {
+        properties.put( tableId, partitionProperty );
+    }
+
+
+
+    @Override
+    public void updatePartition( long partitionId, Long partitionGroupId ) {
+
+    }
+
+
+    @Override
+    public AllocationPartition addPartition( long namespaceId, long adapterId, long tableId, PlacementType placementType, DataPlacementRole role ) {
+        /*long id = idBuilder.getNewPartitionId();
+        AllocationPartitionOld partition = new AllocationPartitionOld(
+                id,
+                namespaceId,
+                tableId,
+                adapterId,
+                placementType,
+                role );
+
+        allocationPartitions.put( Pair.of( adapterId, id ), partition );
+        return partition;*
+         */
```

Comment:
Since the tests are not failing, this method seems to be an artifact.

Reply:
Removed.